subsystems/build.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  init: require('bare-dev/init'),
```

Comment:
are these explicitly marked as public apis by bare-dev or is this a workaround of missing API?

Reply:
Yup. These are directly exported by bare-dev in its package.json